README.md

```diff
@@ -32,11 +32,13 @@ machine learning interatomic potentials aiida plugin
 The code relies heavily on [janus-core](https://github.com/stfc/janus-core), which handles mlip calculations using ASE.
 
 
+# Getting Started
+
 ## Installation
+Create a Python [virtual environment](https://docs.python.org/3/library/venv.html#creating-virtual-environments) and activate it to install aiida-mlip
 
 ```shell
 pip install aiida-mlip
-verdi quicksetup  # better to set up a new profile
 verdi plugin list aiida.calculations
 ```
 The last command should show a list of AiiDA pre-installed calculations and the aiida-mlip plugin calculations (mlip.opt, mlip.sp)
```

Comment:
```suggestion
The last command should show a list of AiiDA pre-installed calculations and the aiida-mlip plugin calculations:
```
I know this isn't new, but not sure these are necessary when we list them all anyway.